core/src/ast/function.rs

```diff
@@ -89,6 +89,7 @@ pub enum Function {
         expr: Expr,
         chars: Option<Expr>,
     },
+    Rand(Vec<Expr>),
```

Comment:
This looks that `RAND` function takes 0 or 1 argument but current ast accepts variable number of `Expr`.

It would be enough with `Rand(Option<Expr>)`

Reply:
Do we need to create a OptionalExpr function or is there a way to use Exprs::Single([expr].into_iter()) ?
Here isthe error I'm getting:

`
 | Self::Rand(expr)
| ^^^^expected enum `ast::expr::Expr`, found enum `std::option::Option`
`

Reply:
I'm still getting this error.   Any way of fixing it?
 --> core\src\plan\expr\function.rs:39:26
   |
17 |         match self {
   |               ---- this expression has type `&ast::function::Function`
18 |             Self::Now() | Function::Pi() | Function::GenerateUuid() => Exprs::Empty(empty()),
19 |             Self::Lower(expr)
   |                         ---- first introduced with type `&ast::expr::Expr` here
...
39 |             | Self::Rand(expr)
   |                          ^^^^ expected enum `ast::expr::Expr`, found enum `std::option::Option`
   |